allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -91,42 +92,140 @@ def __init__(
         detokenized = " ".join(tokenized)
         self._tokenizer_lowercases = "a" in detokenized
 
-        self._add_special_tokens = add_special_tokens
-        self._max_length = max_length
-        self._stride = stride
-        self._truncation_strategy = truncation_strategy
-        self._calculate_character_offsets = calculate_character_offsets
+        # Reverse-engineer the tokenizer for two sequences
```

Comment:
Maybe split this into a separate method?

Reply:
I looked into doing this, and I don't like what it becomes. We'd have to return 5 values from the method, and pass in two tokenizers. There is no way to get a tokenizer with special tokens from one without special tokens, and the function needs both. It's all solvable, but not worth it.